packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -2911,3 +2967,5 @@ function nestedContext(
         : context.surroundingTypeFqns,
   };
 }
+
+const isDeprecated = (x: PythonBase) => x.docs?.deprecated !== undefined;
```

Comment:
I'd just add `isDeprecated` on `PythonBase` or something.

Reply:
`PythonBase` is an interface 😭